flutter/shell/platform/tizen/flutter_tizen_view.cc

```diff
@@ -49,9 +49,8 @@ namespace flutter {
 FlutterTizenView::FlutterTizenView(std::unique_ptr<TizenViewBase> tizen_view)
     : tizen_view_(std::move(tizen_view)) {
   tizen_view_->SetView(this);
-
```

Comment:
Removing this blank line will make the code less readable.

Reply:
done